### ISSUE TITLE
Support pure Java Graphviz engine

### ISSUE THREAD

**Issue by schemacrawler** [schemacrawler/SchemaCrawler#163]
Support pure Java Graphviz engine, so that Graphviz does not need to be installed
https://github.com/nidi3/graphviz-java

**Comment by sualeh** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Support put Java Graphviz engine](https://trello.com/c/XDODl6G2/350-support-put-java-graphviz-engine)

**Comment by sualeh** [comment]
```java
Graphviz
      .fromFile(new File(dotFile))
      .width(700)
      .render(Format.SVG)
      .toFile(new File(outputFile));
```

**Comment by adriens** [comment]
WOW, thanks for pointing this java port !!! It opens a wide set of possibilities around graphs. For now, i could ony user Gephi sdk to draw complexe graphs i pure java.

**Comment by sualeh** [comment]
Fixed in 14.20.01
